src/main/java/cz/cvut/kbss/termit/dto/statistics/TermFrequencyDto.java

```diff
@@ -0,0 +1,79 @@
+package cz.cvut.kbss.termit.dto.statistics;
+
+import cz.cvut.kbss.jopa.model.annotations.*;
+import cz.cvut.kbss.jopa.vocabulary.RDFS;
+
+import java.io.Serializable;
+import java.net.URI;
+import java.util.Objects;
+
+@SparqlResultSetMapping(name = "TermFrequencyDto", classes = {
+        @ConstructorResult(targetClass = TermFrequencyDto.class,
+                variables = {
+                        @VariableResult(name = "vocabulary", type = URI.class),
+                        @VariableResult(name = "count", type = Integer.class),
+                        @VariableResult(name = "label", type = String.class)
+                })})
+@OWLClass(iri = "http://onto.fel.cvut.cz/ontologies/application/termit/pojem/term-frequency")
```

Comment:
nemame to v nejakych konstantach?

Reply:
To těžko, tenhle typ vlastně neexistuje, vymyslel jsem si ho, aby ten objekt šel serializovat do JSON-LD.